CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.0.0-rc.0 (2020-02-18)
+
+### New features
```

Comment:
Do we need a header like this when we don't have a corresponding header for breaking changes?

Reply:
I'm torn. I think it would be nice to use *New features*, *Bug fixes*, and *Breaking changes* headings going forward, but I'll leave that up to you.

Reply:
I don't have any objection to using those going forward. I'll leave it as is. ... Do you think we should run this by the rest of the team so that we follow this consistently across changelogs?